actionview/lib/action_view/template.rb

```diff
@@ -113,6 +113,11 @@ class Template
 
     attr_reader :source, :identifier, :handler, :original_encoding, :updated_at
 
+    RUBY_KEYWORDS = %w(begin end alias and begin break case class def do else
+                       elsif end ensure false for if in module next nil not or
+                       redo rescue retry return self super then true undef
+                       unless until when while yield).to_set
```

Comment:
I think we already have a list like this.. AS delegate, maybe?

Is there a practical way we can re-use it? Possible it makes more sense to keep a separate copy here, but let's at least explore our options.